### PR TITLE
how do I inject BlocChatCookies into my home controller?

### PR DESCRIPTION
**Pull request by javierforero**
Hi @droberts84,

please see `HomeCtrl.js` to see how I am injecting it. I know I can inject `$cookies` and get the current user from the home controller, but I figured I'd use the BlocChatCookies service since its already in there.

### CODE REVIEW

File: app/scripts/controllers/HomeCtrl.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 (function(){
-  function HomeCtrl(Room, Message, BlocChatCookies, $uibModal) {
+  function HomeCtrl(Room, Message,$cookies, $uibModal) {
 
     this.all = Room.all;
     this.currentRoom = null;
+    this.currentUser = $cookies.get('blocChatCurrentUser');
```
Comment (drobtravels) [inline app/scripts/controllers/HomeCtrl.js]: Note that this code will run as soon as `HomeCtrl` launches and "cache" the value in `this.currentUser`.  Try clearing your cookies, then setting the user name via modal.  What is `this.currentUser` equal to when you send a message?

**Comment by javierforero** [comment]
@droberts84 see "finished chat project" commit for any further feedback. Thanks for helping me with this

File: app/scripts/controllers/HomeCtrl.js [hunk 2]
```diff
@@ -24,14 +23,15 @@
     };
 
     this.send = function() {
-
+      var myUser = $cookies.getObject('blocChatCurrentUser');
+      console.log(myUser);
```
Comment (drobtravels) [inline app/scripts/controllers/HomeCtrl.js:27]: probably want to remove this

**Review by drobtravels — APPROVED** [review]
No problem, GREAT work on this!  You've put together a real live working chat room.

**Pull request merged**